packages/esm-patient-search-app/src/patient-search-icon/patient-search-icon.component.scss

```diff
@@ -55,3 +55,11 @@ $inverse-02:#393939;
 .searchButton {
   background-color: $inverse-02;
 }
+
+.bx--search .bx--search-close::before{
+  content: "Clear";
+}
+
+.bx--search .bx--search-close svg{
+  display: none;
+}
```

Comment:
@ibacher atleast plain css had to do the trick but it has also failed.  Do you have any other options i should use?

Reply:
Thank you, @ibacher 
I think `Internal` are accessible through released carbon packages 

I am thinking of a way we can do some sort of extending the Search-Component and override the returned/rendered fields  
Is this possible?

Reply:
I don't think it would be worth it to fork the Search component just for this.

My suggestion: create an element containing the text and use CSS to hack it into place. It will be hacky but I think it is the best way forward. `display` and probably `position` (or maybe `margin`) will be the key CSS properties to mess with here.

Reply:
I'm actually thinking we maybe just leave the X and do what @jnsereko already did to have a label saying "Clear"